docs/source/global_settings.rst

```diff
@@ -0,0 +1,21 @@
+Global Settings
+================
+
+The global_settings.py module is the home to the global variable GAMEBENCH_CONFIG.
+This dictionary holds the API endpoint information that the library uses.
+It also contains the username and password for the GameBench account you are using,
+and the company ID.
```

Comment:
I didn't know how to get the company ID until I asked you, Tristan. I think it may be worth mentioning where to get it so that the engineer won't have to leave the docs to search for where they might find the location.

Reply:
Good call!  I've updated the global settings section with a few sentences on finding the company id.

Reply:
Awesome :) Thanks!

Reply:
Added information on finding the company id.